rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -1797,6 +1804,18 @@ error_code sys_net_bnet_getsockname(ppu_thread& ppu, s32 s, vm::ptr<sys_net_sock
 
 			return {};
 		}
+		#ifdef _WIN32
+		else
+		{
+			// windows doesn't support getsockname for sockets that are not bound
+			if (get_native_error() == 10022)
+			{
+				memset(&native_addr, 0, native_addrlen);
```

Comment:
Horrible hack, you can just save it in IDM instead of the operating system I think.

Reply:
I agree that windows is a horrible hack but I don't understand what you want to save in IDM?

Reply:
sockname, dont rely on windows for it. unless it is expected to be empty for unbound sockets?

Reply:
yes it's expected to be 0.0.0.0:0 if unbound